conversion/conversion.go

```diff
@@ -587,6 +611,25 @@ func WriteSessionFile(conv *internal.Conv, name string, out *os.File) {
 	fmt.Fprintf(out, "Wrote session to file '%s'.\n", name)
 }
 
+// WriteDownloadableFiles creates a directory labeled downloads with the current timestamp
+// where it writes the sessionfile, report summary and DDLs then returns the directory where it writes.
+func WriteDownloadableFiles(conv *internal.Conv, dbName string, driver string, BytesRead int64, out *os.File) (string, error) {
+	now := time.Now()
+	dirPath := "frontend/harbour_bridge_downloads/" + dbName + "/"
```

Comment:
We can skip the prefix "harbour_bridge" since frontend/ folder will always be inside harbourbridge repo. 

Also I wonder what is the behavior when instead of running harbourbridge from the repo, a pre-built harbourbridge is used? Is it better to write at the current working directory?